build/buildpipeline/pipeline.groovy

```diff
@@ -12,7 +12,7 @@ def parameters = [
 windowsPipeline.triggerPipelineOnEveryGithubPR("Windows ${configuration} x64 Build", parameters)
 windowsPipeline.triggerPipelineOnGithubPush(parameters)
 
-windowsESPipeline.triggerPipelineOnEveryGithubPR("Windows ${configuration} x64 Build (Spanish language image) ", parameters)
+windowsESPipeline.triggerPipelineOnEveryGithubPR("Windows ${configuration} (Spanish Language) x64 Build", parameters)
```

Comment:
I'd remove the parantheses to be extra safe. Unfortunately the name change won't show up until it's merged in to dev

Reply:
> Unfortunately the name change won't show up until it's merged in to dev

Yep, I was expecting that.